roadmap/2022.md

```diff
@@ -0,0 +1,57 @@
+# Roadmap 2022
+
+This is the roadmap planned by the Integration team at MeiliSearch for the year 2022.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2022: the Main Lines
+
+#### Projects
+
+- Better know our users and our community to ease our decision making
+  - set the analytics on every integration
+  - Extract information from them to prioritize our tasks
+- Keep working with the community, improve our way of working with the contributors by creating clearer issues, and by providing more and more "good first issues".
+- Work with the devrel team to have help for the support for answering issues. The goal is to allow the integration team developers to focus on the PR reviews and their coding tasks, rather than doing support.
+- Work with the documentation team to provide complete and easy-to-maintain documentation for each SDK
+- Be more organized regarding our rhythm of working
+    - during the pre-release period: improve the process and define steps for each sprint
+    - during the implementation period: find the right balance between maintainance and new content addition.
+- Release our first database integrations since we don't provide any in 2021.
+
+#### Human resources
+
+- New manager welcoming
+- Recruitment of new developers
+
+## Q1
+
+Database:
+- [ ] Finish Firebase integration
+- [ ] Write the "Firebase + MeiliSearch" article
+
+Strapi:
+- [ ] Make Strapi plugin compatible with the new Strapi MarketPlace. Will depend on the Strapi's docs.
+- [ ] (?) Write article "how to make your plugin compatible with the new Strapi MarketPlace"
+- [ ] Participate to the Strapi conference in March
+
+Back-end web framework:
+- [ ] Start Laravel integration containing the missing features in [Laravel/Scout](https://github.com/laravel/scout)
+
+Front-end
+- [ ] Fix the pagination issues in `instant-meilisearch`
+- [ ] First release of Gatsby
+- [ ] (?) Write the "Gatsby + MeiliSearch" article, depending on the @mdubus availability
```

Comment:
sorry I commented this in the 2021 file, @shivaylamba is on it

Reply:
I've already removed it :) thanks!